src/App.js

```diff
@@ -32,15 +33,69 @@ function App() {
       job: "github",
     },
   };
+  const [apiData, setApiData] = useState(UserData);
 
   const ContactIcons = [faLocationDot, faTwitter, faLink, faBuilding];
 
+  const searchHandler = async (username) => {
```

Comment:
Name this as `fetchUser` It is more descriptive this way. Explains what the method is doing which is fetching a user. Handler methods are passed directly to the events they are meant to handle i.e `onClick`. This is not being passed directly

```suggestion
  const fetchUser = async (username) => {
```